lib/ansible/modules/cloud/cloudstack/cs_resourcelimit_facts.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016, René Moser <mail@renemoser.net>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['stableinterface'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: cs_resourcelimit_facts
```

Comment:
the community has decided that modules should be named "_facts" when providing host facts, otherwise the sufix should be _info. That is why this module should be renamed `cs_resourcelimit_info`